template.yml

```diff
@@ -236,6 +251,9 @@ Resources:
               runCommand:
                 # Ensure that if a command fails the script does not proceed to the following commands
                 - "$ErrorActionPreference = \"Stop\""
+                - "$ENV:PipelineExecutionId = \"{{ executionId }}\""
```

Comment:
I think we need to add these variable to _linux_script_ step below as well to ensure we have a consistent experience across platforms.